### PR TITLE
Support bridge set fee

### PR DESCRIPTION
**Pull request by tolak**

### CODE REVIEW

**Comment by tolak** [comment]
Looks like we have replaced deposit_event with push_message to send message to TEE which case the conflicts.

**Comment by h4x3rotab** [comment]
`push_message` means to send something to Confidential Contract. In this case you just want to emit a message for the user, not for Confidential Contract. So we should still use the original `deposit_event`.

Now you have implemented a flat fee model. I asked the product manager @DavyChang  to arrange a discussion with you about the fee model. He is in charge of the design.

File: pallets/bridge_transfer/src/lib.rs [hunk 1]
```diff
@@ -34,8 +33,8 @@ pub enum LotteryEvent {
 	OpenBox(u32, u32, Vec<u8>),
 }
 
-pub trait Config: system::Config + bridge::Config + pallet_mq::Config {
```
Comment (kvinwang) [inline pallets/bridge_transfer/src/lib.rs:37]: Why the pallet_mq::Config be removed?

Comment (tolak) [reply pallets/bridge_transfer/src/lib.rs:37]: It was removed accidental, already fixed.

File: pallets/bridge_transfer/src/lib.rs [hunk 2]
```diff
@@ -74,7 +82,18 @@ decl_module! {
...
+        fn deposit_event() = default;
+
+        /// Change extra bridge transfer fee that user should pay
+		#[weight = 195_000_000]
+		pub fn sudo_change_fee(origin, amount: BalanceOf<T>, dest_id: bridge::ChainId) -> DispatchResult {
+			ensure_root(origin)?;
+			BridgeFee::<T>::insert(dest_id, amount);
+			Self::deposit_event(RawEvent::FeeUpdated(dest_id, amount));
+			Ok(())
+        }
```
Comment (kvinwang) [inline pallets/bridge_transfer/src/lib.rs:96]: indentation

**Comment by kvinwang** [comment]
LGTM once the indentation be fixed.

File: pallets/bridge_transfer/src/lib.rs (hunk 2 above)
Comment (tolak) [reply pallets/bridge_transfer/src/lib.rs]: Fixed

**Comment by h4x3rotab** [comment]
Thanks @kvinwang for reviewing. I think we are good to merge the change.

@tolak Please note that we are still working on designing the fee model. Will let you know once we got the concrete requirement.

**Comment by tolak** [comment]
@h4x3rotab Got it

**Review by h4x3rotab — APPROVED** [review]

**Pull request merged**